cmd/buildctl/build.go

```diff
@@ -291,6 +297,20 @@ func buildAction(clicontext *cli.Context) error {
 		// not using shared context to not disrupt display but let is finish reporting errors
 		return progressui.DisplaySolveStatus(context.TODO(), "", c, os.Stderr, displayCh)
 	})
+	err = eg.Wait()
```

Comment:
assign it to `err` because it will be checked in `defer`

Reply:
Perhaps a valid reason to use a named return variable, to make these more obvious and prevent someone accidentally "masking" the variable i.e.

```go
func buildAction(clicontext *cli.Context) (retErr error) {
```

Reply:
Thanks @thaJeztah I'll update~